wordle-web/src/components/SettingsMenu.vue

```diff
@@ -35,7 +35,7 @@ function setOceanTheme() {
           </v-toolbar-title>
         </v-toolbar>
         <v-card-text>
-          <V-text class="text-h5"> Theme: </V-text> <br /><br />
+          <v-card-title class="text-h5 pb-8"> Theme: </v-card-title>
```

Comment:
There is no `v-text` tag from Vuetify, I replaced it here with the `v-card-title` tag and added some bottom padding to space it out from the theme buttons below